docs/advanced/pdf.md

```diff
@@ -107,3 +110,52 @@ jb build mybookname/ --builder latex
 ```
 
 ::::
+
+
+### Updating the name of the PDF file
+
+To update the name of your `PDF` file you can set the following in `_config.yml`
+
+```yaml
+latex:
+  latex_documents:
+     targetname: book.tex
+```
+
+This will act as an automatic `override` when sphinx builds the
+[latex_documents](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-latex_documents). It is typically inferred by `sphinx` but when
+using `jupyter-book` naming the file in the `_config.yml` generally makes it
+easier to find.
+
+### Using a different LaTeX engine
+
+Some users may want to switch to using a different LaTeX engine to build the
+`PDF` files. For example, if your project contains `Unicode` you will need to
+use `xelatex` to build the `PDF` file.
+
+To update the `LaTeX` engine to `xelatex` you can add the following to your `_config.yml`
+
+```yaml
+latex:
+  latex_engine: xelatex
+```
+
+:::{note}
+We will be making `xelatex` the default in the near future, so this can be used to
+specify other builders such as `pdflatex`, or `lualatex`.
+
+See the Sphinx documentation [for available builders](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-latex_engine)
```

Comment:
can you standardize on capitalized Sphinx ? (you use lowercase sphinx elsewhere).